prov/gni/include/gnix_ep.h

```diff
@@ -115,4 +193,33 @@ typedef ssize_t (*trecvmsg_func_t)(struct fid_ep *ep,
 				   const struct fi_msg_tagged *msg,
 				   uint64_t flags);
 
+/*
+ * inline functions
+ */
+
+static __attribute__((unused)) struct gnix_fab_req *
```

Comment:
if I didn't include unused for a couple of files I'd get a compiler warning about static function being unused.

for the ```inline``` maybe just drop and let the compiler try it?  I notice ```inline``` is used in quite a few places in libfabric.